### ISSUE TITLE
Is tiles api the processing api ?

### ISSUE THREAD

**Issue by tik65536** [LandscapeGeoinformatics/pydggsapi#4]
The tiles api is served for MVT purpose which you send the layer name with the x,y,z, then it returns the reuqired data for the extent.
This come up with a question, what is the layer name defining? 

The layer name may serve for 2 purposes: 
1. The collection(s) , aka, the data 
2. The operation on the data to create new variable(s) to return

So, does tiles api got the role of processing ?

**Comment by allixender** [comment]
No. Tiles is Tiles and shall only deliver hexagon geometries with associated data.
